content/influxdb/v2.0/telegraf-configs/update.md

```diff
@@ -72,16 +73,3 @@ influx telegrafs update \
   -d "This is a description for an example Telegraf configuration."
   -f /path/to/telegraf.conf
 ```
-
-## Customize an existing Telegraf configuration
```

Comment:
I think we can still keep this section but rename the heading (something like "Download your existing Telegraf configuration"?? you can probably come up with something better). Since you can still download your config and there might be a reason people want to download it, edit it on their local machine, and re-upload it back up.

Reply:
We had this as kind of a workaround because there was no UI option. It says elsewhere that you can download the config so I don't think we need this set of steps anymore.